src/Less5Collections/phonebook/Contact.java

```diff
@@ -1,20 +1,19 @@
 package Less5Collections.phonebook;
 
-public class Notizen {
+public class Contact {
     private final String name;
     private final String[] telephone;
```

Comment:
one contact should contain only one number. PhoneBook can have many contacts (with name duplicates)